src/antsibull/cli/antsibull_build.py

```diff
@@ -370,10 +374,29 @@ def parse_args(program_name: str, args: list[str]) -> argparse.Namespace:
     validate_deps.add_argument('collection_root',
                                help='Path to a ansible_collections directory containing a'
                                ' collection tree to check.')
+    validate_tags_shared = argparse.ArgumentParser(add_help=False)
+    validate_tags_shared.add_argument(
+        '-I', '--ignore',
+        action='append', help='Ignore these collections when reporting errors.'
+    )
+    validate_tags_shared.add_argument(
+        '--ignores-file',
+        help='Path to a file with newline separated list of collections to ignore',
+        type=argparse.FileType('r'),
+    )
+    validate_tags_shared.add_argument(
+        '-W', '--warn-useless-ignores',
```

Comment:
It's a bit strange to call this "warn" when it is in fact an error. How about calling this `--fail-on-useless-ignores` or something like that?

Reply:
Actually, I'd prefer `error_on_useless_ignores`.

Reply:
SGTM :)